zproject/computed_settings.py

```diff
@@ -1244,6 +1244,28 @@ def ensure_dict_path(d: dict[str, Any], keys: list[str]) -> None:
 EMAIL_GATEWAY_PASSWORD = get_secret("email_gateway_password")
 AUTH_LDAP_BIND_PASSWORD = get_secret("auth_ldap_bind_password", "")
 
+########################################################################
+# LiteLLM SETTINGS
+########################################################################
+
+# The model name that will used by LiteLLM library to configure
+# parameters to be sent to API.
+# The Llama-3-8B-instruct model is free to use and only requires submitting
+# a small form on the HuggingFace page for the model to gain access.
+# We only support HuggingFace and AWS Bedrock for LLM API requests.
+TOPIC_SUMMARIZATION_MODEL = get_secret(
+    "topic_summarization_model", "huggingface/meta-llama/Meta-Llama-3-8B-Instruct"
+)
```

Comment:
We want this to default to `None` if not specified in production, and the view code should check for that and return a user-facing JsonableError of the form "AI features are not enabled on this server."

Reply:
Makes sense. Done.